feat: add typegen and docs for RawSlice, Bytes, and StdString (#1342)
FuelLabs/fuels-ts: apps/docs/src/guide/types/bytes.md

# Bytes

> A dynamic array of byte values can be represented using the `Bytes` type, type used to represent raw bytes.

Reviewer: ```suggestion
A dynamic array of byte values can be represented using the `Bytes` type, which represents raw bytes.
```
